### PR TITLE
Add optional troubleshooting step for winget installs

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/2724

### CODE REVIEW

**Review by lzdanski — COMMENTED** [review]
The changes looked good in preview, but I double checked the slack thread and forum, and this change didn't resolve the user's issue. Need to confirm that the change still needs to be made.

File: astro/cli/install-cli.md [hunk 1]
```diff
@@ -110,13 +110,15 @@ By default, the Astro CLI uses Docker as its container management engine for run
     
     From the text that appears, copy the path for the Astro CLI executable. It should be similar to `C:\Users\myname\AppData\Local\Microsoft\WinGet\Packages\Astronomer.Astro_Microsoft.Winget.Source_8wekyb3d8bbwe`. 
 
-3. Paste the path into File Explorer or open the file path in terminal, then rename the Astro executable to `astro.exe`.
+3. Paste the path into File Explorer or open the file path in terminal, then rename the Astro executable to `astro`.
```
Comment (jwitz) [inline astro/cli/install-cli.md:113]:
```suggestion
3. Paste the path into File Explorer or open the file path in terminal, then rename the Astro executable to `astro.exe`.
```

File: astro/cli/install-cli.md [hunk 2]
```diff
@@ -272,7 +274,7 @@ winget uninstall -e --id Astronomer.Astro
 
 To uninstall the Astro CLI on Windows:
 
-1. Delete the filepath for `astro.exe` from your Windows PATH environment variable.
+1. Delete the filepath for `astro` from your Windows PATH environment variable.
```
Comment (jwitz) [inline astro/cli/install-cli.md:277]:
```suggestion
1. Delete the filepath for `astro.exe` from your Windows PATH environment variable.
```

**Review by lzdanski — APPROVED** [review]
Saw follow up slack conversation. This note LGTM

**Pull request merged**